samples/kotlin-sample/src/main/java/com/apollographql/apollo/kotlinsample/data/ApolloCallbackService.kt

```diff
@@ -17,11 +17,11 @@ import com.apollographql.apollo.kotlinsample.type.RepositoryOrderField
  */
 class ApolloCallbackService(apolloClient: ApolloClient) : GitHubDataSource(apolloClient) {
   override fun fetchRepositories() {
-    val repositoriesQuery = GithubRepositoriesQuery.builder()
-        .repositoriesCount(50)
-        .orderBy(RepositoryOrderField.UPDATED_AT)
-        .orderDirection(OrderDirection.DESC)
-        .build()
+    val repositoriesQuery = GithubRepositoriesQuery(
+        50,
```

Comment:
Would it be better to use the named arguments here: Like `repositoriesCount = 50`

Reply:
And same in other places similar to this.